### ISSUE TITLE
Add checksums for macOS 10.15.3 (19D2064) and macOS 10.15.4 (19E266)

### ISSUE THREAD

**Issue by vitalys** [notpeter/apple-installer-checksums#74]
There are new builds

**Comment by xingheng** [comment]
Someone could help me verify this?

macOS 10.15.4
```
48d96f46663e87ce6ca092ce21fd5ce40d84481b  /Applications/Install macOS Catalina.app/Contents/SharedSupport/BaseSystem.dmg
0f553218733618b8b22ad89f680dd280d0251441  /Applications/Install macOS Catalina.app/Contents/SharedSupport/InstallESD.dmg
```

**Comment by vitalys** [comment]
@xingheng I can confirm these checksums

**Comment by sleeve** [comment]
Fixed with af9a7b9895f754c4dcb03e43a0d4d0542cc0a316 and #76.
